src/Builder.ts

```diff
@@ -257,4 +259,16 @@ export class QueryBuilder<GenericResult, GenericResultOne> {
 
     return ` RETURNING ${value.join(', ')}`
   }
+
+  _subquery(value?: string | Subquery): string {
+    if (!value) return ''
+
+    if (typeof value === 'string') return ` (${value})`
```

Comment:
if you agree with the change above, it would also require change here to apply the alias to simple joins

Reply:
That was a good idea for a change. I refactored a bit to move the `alias` to the `Join` interface. It did make the `_join` function a bit hard to read, so if you have thoughts about that, I'm open to suggestions.